bin/create_yml.pl

```diff
@@ -413,16 +430,8 @@ sub get_genelist {
     while (<JSON>) {
         $data = decode_json($_);
     }
-   # print Dumper($data);
     foreach my $key (@{$data}) {
-        if (ref $key->{institute} eq 'ARRAY') {

```

Comment:
Hmm, tricky to follow here. Guessing you know this chunk well enough, so that no new issues are introduced here ...

Reply:
Yes, this is some really old legacy code. It's from the time we had a manually cured list of genepanels. Now it was just confusing to keep